.nvmrc

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
What if this is set to `lts/*`? Remix should always work with current LTS, right?

Reply:
Not sure what's the team policy here. Any thoughts @brookslybrand ?